src/components/Home.jsx

```diff
@@ -41,16 +42,18 @@ export default function Home() {
           {reviews.map((review) => {
             return (
               <li className="review-card" key={review.created_at}>
-                <h3>{review.title}</h3>
-                <p>{review.designer}</p>
+                <Link to={`/reviews/${review.review_id}`}>
+                  <h3>{review.title}</h3>
+                </Link>
+                <p>Designer: {review.designer}</p>
                 <img
                   src={review.review_img_url}
                   alt={review.title}
-                  width="100"
-                  height="100"
+                  width="200"
+                  height="200"

```

Comment:
Not a massive issue, but mixing inline and external css styling can end up looking a bit messy.  Up to you how you carry on with it, but make sure that you are doing things consistently throughout.  If you want to for example carry on doing height/widths inline so you can programmatically change things then go for it, but keep doing that consistently!

Reply:
Noted will look to move all styling across to css once basic functionality is setup :)